### PR TITLE
A PR is approved only if all reviews are approved.

### LINKED ISSUE #40: Only take into account latest reviews
if someone approves, then request changes, then approves, it looks like it's still counts as approved

### PR DESCRIPTION
**Pull request by JulienPalard**
Closes #40

### CODE REVIEW

File: pydocteur/pr_status.py
Comment (Seluj78) [inline pydocteur/pr_status.py:45]: Yes but no :P

I had an `all` before, but the problem is that `get_reviews()` returns also stale reviews, so this will return all reviews, current and old ones, to it will never be merged.

What we need to do is get only the latest and not stale review from each reviewer maybe use some stuff from https://pygithub.readthedocs.io/en/latest/github_objects/PullRequestReview.html

**Review by Seluj78 — CHANGES_REQUESTED** [review]

File: pydocteur/pr_status.py
Comment (JulienPalard) [reply pydocteur/pr_status.py:45]: Fixed in #45 with another approch.

**Pull request closed without merging**
